_i18n/de/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,167 @@
 
-6. You may have to hit confirm twice before it proceeds.
+6. Eventuell musst du zweimal bestätigen, bevor fortgefahren wird.
 
-7. Your Ledger Monero wallet will now be generated. Note that this may take up to 5-10 minutes. Furthermore, there will be no immediate feedback in the CLI nor on the Ledger.
+7. Die Erstellung deines Monero-Ledger-Wallets wird nun fertiggestellt. Beachte, dass dies fünf bis zehn Minuten dauern kann. Es wird außerdem keine direkte Rückmeldung, weder im CLI noch im Ledger, geben.
 
-8. `monero-wallet-cli` will start refreshing. Wait until it has fully refreshed.
+8. `monero-wallet-cli` startet nun den Aktualisierungsvorgang. Warte, bis dieser vollständig abgeschlossen ist.
 
-Congratulations, you can now use your Ledger Monero wallet in conjunction with the CLI.
+Glückwunsch, du kannst dein Monero-Ledger-Wallet nun in Verbindung mit dem CLI verwenden!
 
-### 4. A few final notes
+### 4. Schlussbemerkungen
 
-1. We'd strongly advise to test the full process first. That is, send a small amount to the wallet and subsequently restore it (using aforementioned guide) to verify that you can recover the wallet. Note that, upon recreating / restoring the wallet, you ought to append the `--restore-height` flag (with a block height before the height of your first transaction to the wallet) to the command in step 3 (Windows), step 5 (Mac OS X), or step 3 (Linux). More information about the restore height and how to approximate it can be found [here](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
+1. Wir empfehlen dringlichst, den gesamten Prozess zunächst zu testen. Dies kannst du machen, indem du einen kleinen Betrag an dein Wallet schickst und es daraufhin (unter Verwendung der vorliegenden Anleitung) umspeicherst. Dadurch überprüfst du, ob du dein Wallet tatsächlich wiederherstellen kannst. Beachte, dass du bei der Neuerstellung/Wiederherstellung des Wallets das `--restore-height`-Flag (mit einer Blockhöhe vor deiner ersten Transaktion auf das Wallet) in Schritt 3 (Windows, Linux) oder Schritt 5 (Mac OS X) anhängen solltest. Weitere Informationen bezüglich der Wiederherstellungshöhe und der Annäherung an diese finden sich [hier](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
```

Comment:
Should find another word for "umspeicherst". I think "wiederherstellst" would be better, but doesn't fit with the next sentence.

Reply:
Maybe like that: 
"Wir empfehlen dringlichst, den gesamten Prozess zunächst zu testen. Dies kannst du machen, indem du einen kleinen Betrag an dein Wallet schickst und es daraufhin (unter Verwendung der vorliegenden Anleitung) wiederherstellst. Dadurch überprüfst du, ob die Wiederherstellung deines Wallet tatsächlich funktioniert."
I cannot find another word/synonym for "wiederherstellen" so there is still the repetition. Maybe like this (as a noun in the next sentence) the repetition doesn't stand out that much?

Reply:
Sounds very good to me now!